src/collective/behavior/amp/tests/test_view.py

```diff
@@ -108,6 +111,21 @@ def test_amp_analytics(self):
         self.assertEqual(
             amp.find('*//amp-analytics/script').text, u'"foo": "bar"')
 
+    def test_related_items(self):
```

Comment:
@rodfersou I would prefer to test against the HTML code rendered as in the other features.